fabric-entity-attribute-api-v1/build.gradle

```diff
@@ -0,0 +1,2 @@
+archivesBaseName = "fabric-entity-attribute-api-v1"
+version = getSubprojectVersion(project, "0.1.0")
```

Comment:
I assume this will be bumped to 1.0.0 during the pre releases

Reply:
You see the added class is `@Deprecated`, so why 1.0.0 if this api is subject to change

Reply:
You didn't read the whole message lol. I said this should be bumped when we reach prereleases where this will be mostly concrete.

Reply:
I still want to use 1.0.0 everywhere, experimental doesn't quite mean the same as wip